internal/generator/fluentd/output/security/security.go

```diff
@@ -64,6 +64,10 @@ func HasBearerTokenFileKey(secret *corev1.Secret) bool {
 	return HasKeys(secret, constants.BearerTokenFileKey)
 }
 
+func HasAWSWebIdentityTokenFilePath(secret *corev1.Secret) bool {
+	return HasKeys(secret, constants.AWSWebIdentityTokenFilePath)
```

Comment:
This key won't be in the secret.   
Need to add and use a new constant AWSWebIdentityTokenKey = "web_identity_token_file"

Reply:
If they are adding it, then this works.